From 50713029d2768778cf531c168e93694b30cb9742 Mon Sep 17 00:00:00 2001
From: raggi <jftucker@gmail.com>
Date: Wed, 29 Apr 2009 00:28:12 +0100
Subject: [PATCH] Switch to emacs style encoding comment

---
 lib/rack/utils.rb |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/lib/rack/utils.rb b/lib/rack/utils.rb
index 3d26849..42e2e69 100644
--- a/lib/rack/utils.rb
+++ b/lib/rack/utils.rb
@@ -1,4 +1,4 @@
-# coding=binary
+# -*- encoding: binary -*-
 
 require 'set'
 require 'tempfile'
-- 
1.6.0.4

